From 041cc342471c626d453797a8ba07c3612e27a074 Mon Sep 17 00:00:00 2001
From: Tao Huang <huangtao@rock-chips.com>
Date: Mon, 2 Nov 2020 09:52:05 +0800
Subject: [PATCH] Revert "ARM: configs: rv1126_defconfig: enable
 ROCKCHIP_BUS_DEVFREQ"

This reverts commit 60005ae6b062ab7469fa4bd16f565852e5034643.
Which make cpu unstable now.

Change-Id: Ib22d3ca75709f810a22d8754aac40cfab120022b
Signed-off-by: Tao Huang <huangtao@rock-chips.com>
---
 arch/arm/configs/rv1126_defconfig | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm/configs/rv1126_defconfig b/arch/arm/configs/rv1126_defconfig
index 2b813d8520c1..b65b5e17ecc2 100644
--- a/arch/arm/configs/rv1126_defconfig
+++ b/arch/arm/configs/rv1126_defconfig
@@ -37,7 +37,6 @@ CONFIG_CPU_FREQ=y
 CONFIG_CPU_FREQ_DEFAULT_GOV_ONDEMAND=y
 CONFIG_CPU_FREQ_GOV_USERSPACE=y
 CONFIG_CPUFREQ_DT=y
-CONFIG_ARM_ROCKCHIP_BUS_DEVFREQ=y
 CONFIG_ARM_ROCKCHIP_CPUFREQ=y
 CONFIG_CPU_IDLE=y
 CONFIG_ARM_CPUIDLE=y
-- 
2.35.3

